webpack/assets/javascripts/react_app/components/BreadcrumbBar/BreadcrumbBarReducer.js

```diff
@@ -27,6 +28,9 @@ export default (state = initialState, action) => {
     case BREADCRUMB_BAR_CLEAR_SEARCH:
       return state.set('searchQuery', '');
 
+    case BREADCRUMB_BAR_UPDATE_TITLE:
+      return state.set('titleAddition', payload);
```

Comment:
Don't we want some `titleAddition` value to be in the `initialState`?

Reply:
isn't it enough have a default value defined in the Proptype?

Reply:
In this case it will achieve the same results, but I wouldn't say it is enough, they are playing different roles.

Different defaultProps when not passing a value vs the actual initial state of the application.